tutorials/tracing.md

```diff
@@ -0,0 +1,189 @@
+configuration, which refers to the amount of samples collected per million
+spans. This means if it being set to `0` (the default), then CRI-O will not
+collect any traces at all. If set to `1000000` (one million), then CRI-O will
+create traces for all created spans. If the value is below one million, then
+there is **no** way right now to select a subset of spans other than the modulo
+of the set value.
+
+[conmon-rs][conmon-rs] has the capability to add additional tracing past the
+scope of CRI-O. This is automatically enabled when the `pod` runtime type is
+chosen, like so:
+
+```toml
+[crio.runtime]
+default_runtime = "runc"
+
+[crio.runtime.runtimes.runc]
+runtime_type = "pod"
+```
+
+[conmon-rs]: https://github.com/containers/conmon-rs
+
+Then conmon-rs will export traces and spans in the same way CRI-O does
+automatically. Both CRI-O and conmon-rs will correlate their logs to the traces
+and spans. If the connection to the OTLP instance gets lost, then CRI-O will not
+block, and all the traces during that time will be lost.
+
+## Usage example
+
+The [OpenTelemetry Collector][collector] alone cannot be used to visualize
+traces and spans. For that a frontend like [Jaeger][jaeger] can be used to
+connect to it. To achieve that, a configuration file for OTLP needs to be
+created, like this `otel-collector-config.yaml`:
+
+[jaeger]: https://www.jaegertracing.io
+[collector]: https://opentelemetry.io/docs/collector
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      http:
+      grpc:
+
+exporters:
+  logging:
+    loglevel: debug
+
+  jaeger:
+    endpoint: localhost:14250
+    tls:
+      insecure: true
+
+processors:
+  batch:
+
+extensions:
+  health_check:
+  pprof:
+    endpoint: localhost:1888
+  zpages:
+    endpoint: localhost:55679
+
+service:
+  extensions: [pprof, zpages, health_check]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [logging, jaeger]
+    metrics:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [logging]
+```
+
+The `jaeger` `endpoint` has been set to `localhost:14250`, means before starting
+the collector we have to start the jaeger instance:
+
+```bash
+podman run -it --rm --network host jaegertracing/all-in-one:1.41.0
+```
+
+After jaeger is up and running we can start the OpenTelemetry collector:
+
+```bash
+podman run -it --rm --network host \
+    -v ./otel-collector-config.yaml:/etc/otel-collector-config.yaml \
+    otel/opentelemetry-collector:0.70.0 --config=/etc/otel-collector-config.yaml
+```
+
+The collector logs will indicate that the connection to Jaeger was successful:
+
+```text
+2023-01-26T13:26:22.015Z        info    jaegerexporter@v0.70.0/exporter.go:184  \
+    State of the connection with the Jaeger Collector backend \
+    {"kind": "exporter", "data_type": "traces", "name": "jaeger", "state": "READY"}
+```
+
+The Jaeger UI should be now available on http://localhost:16686
```

Comment:
IDK, do we usually backtick url examples like this one?

Reply:
I don't think we have to, but I'm not strongly opinionated on it.

Reply:
I'm on the fence too, it was more of a precedence question.  If you do it elsewhere, I'd backtick it here too.

Reply:
I agree, going for the backticks.